sessions/TypeScript-100/common-types-examples.ts

```diff
@@ -0,0 +1,150 @@
+const isFun: boolean = false;
+
+// there is no int or float, everything is number to JS/TS
+const pie: number = 3.14;
+
+// strings can use single or double quotes
+const fullName: string = "Bob Bobbington";
+
+// template string, using backticks
+const greet: string = `Salutations ${fullName}!`
+
+// symbol
+// note that both variables have what seems to be the same value
+const firstName = Symbol("name");
```

Comment:
you could strongly type the `firstName` declaration with `symbol` to demonstrate the difference between the `symbol` type and the `Symbol` constructor?

```suggestion
const firstName: symbol = Symbol("name")
 ```

Reply:
Basically thinking it would be good to guide away from the `don'ts` :)
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html